files.php

```diff
@@ -256,12 +257,12 @@ function find_genome($path){
 					$genome = '';
 					// Find genome from BAM or SAM files
 					if(substr($path, -4) == '.bam' || substr($path, -4) == '.bam'){
-						$bam_header = shell_exec (escapeshellcmd ('samtools view -H '.$path));
+						$bam_header = shell_exec (escapeshellcmd ('/storage/Software/packages/samtools-1.3.1/bin/samtools view -H '.$path));
```

Comment:
Hmm, it's not super easy no. You can't call system commands within PHP to do it, as this creates a new session for the command which doesn't affect PHP itself which is already running. Easiest way is just to do the `module load` statements before starting the Labrador I guess?

Otherwise, this is a situation where you can use a config variable which is a blank string by default for most users, but can be set to a path for your environment. All that the `module load` commands do is to add a directory to your `PATH`, so then you can use this variable to just hardcode the directory as you're doing above.

I don't feel like I've been very clear there. Does that make sense?